db/migrate/20230713230334_journal_add_keywords_to_entries.rb

```diff
@@ -0,0 +1,5 @@
+class JournalAddKeywordsToEntries < ActiveRecord::Migration[7.0]
+  def change
+    add_column :journal_entries, :keywords, :string, array: true
```

Comment:
I'm surprised these are not tied in any way to the ids of the Journal keywords -- I would have expected you'd want that to show "all entries with keyword X in Journal".

Reply:
You know, I was thinking a join table would be "too much" but I suppose it's not?

The way I was planning to find the `Entries` was indexing this keywords field and using the scope you suggested above; but I do worry that that would make the relationy bits a bit less railsy